app/queries/admin/moderators_query.rb

```diff
@@ -4,9 +4,7 @@ class ModeratorsQuery
       state: :trusted
     }.with_indifferent_access.freeze
 
-    # TODO: [@jacobherrington] Once all Forems have been deployed and the data update scripts have
```

Comment:
The roles have been removed, so we can also delete them from the array and remove this comment.